core/models/asset.py

```diff
@@ -576,16 +576,43 @@ def _save_initial_incident_report_state(self):
         existing_state = StateTransition.objects.filter(asset_incident_report=self)
         if not existing_state:
             StateTransition.objects.create(
-                asset_incident_report=self, state=constants.NEWLY_REPORTED
+                asset_incident_report=self,
+                incident_report_state=constants.NEWLY_REPORTED,
             )
-            self.save()
+
+        # user updates asset_status if lost
+        if self.incident_type == "Loss":
+            asset_status = AssetStatus.objects.filter(asset_id=self.asset.id).first()
+            asset_status.current_status = "Lost"
+            asset_status.save()
 
 
 class StateTransition(models.Model):
     asset_incident_report = models.ForeignKey(
         'AssetIncidentReport', on_delete=models.PROTECT
     )
-    state = models.CharField(max_length=50, default=constants.NEWLY_REPORTED)
+    incident_report_state = models.CharField(
+        max_length=50,
+        choices=constants.REPORT_STATE_OPTIONS,
+        default=constants.NEWLY_REPORTED,
+    )
+    asset_state = models.CharField(
```

Comment:
we can work with the AssetStatus model we currently have

Reply:
If I'm to recall the sync I had with Madge, we are to only update the asset's state (in the AssetStatus model), if damaged is passed. This leaves the other options for the asset (requires repair and requires external assessment) with a need of a field to be placed in. And so this additional field was created. A note was also added in the PT story regarding this.

Reply:
oh.. cool. could we give it a name that won't be confused with the [asset status](https://github.com/AndelaOSP/art-backend/blob/ab3a9ed77f0a92cdc9cf870956bce193dd28e768/core/constants.py#L18)?

Reply:
I have updated it to `asset_state_from_report`.